app/views/govuk_publishing_components/components/_step_by_step_nav.html.erb

```diff
@@ -34,12 +34,12 @@
           circle_class = "gem-c-step-nav__circle--logic" if logic
         %>
         <li class="gem-c-step-nav__step js-step
-          <%= "gem-c-step-nav__step--optional" if step[:optional] %>
           <%= "gem-c-step-nav__step--active" if step_is_active %>"
           <% if step_is_active %>aria-current="step"<% end %>
           <%= "data-show" if step_count == show_step %>
           id="<%= id %>"
           data-track-count="stepNavSection"
+          <%= "data-optional" if step[:optional] %>
```

Comment:
Can `data-optional` be without a parameter?

Reply:
Yes, I believe so. It's used as a true/false value: true if present, false if not.

Reply:
Interesting, I assumed it would be invalid without a parameter. But I just checked this in a validator and you're right, it's fine.